tsdb/head_test.go

```diff
@@ -4007,24 +4008,44 @@ func TestSnapshotError(t *testing.T) {
 	require.NoError(t, err)
 	f, err := os.OpenFile(path.Join(snapDir, files[0].Name()), os.O_RDWR, 0)
 	require.NoError(t, err)
-	_, err = f.WriteAt([]byte{0b11111111}, 18)
+	// lets corrupt middle of the snapshot, so we can replay some entries
+	_, err = f.WriteAt([]byte{0b11111111}, 300)
```

Comment:
Is the change from 18 to 300 significant?

Reply:
Yeah.. it is..

If we corrupt the byte 18, we will not be able to restore any series (and so we cannot see the problem).

Corrupting the byte 300, we are able to restore 2 timeseries before reaching the corrupted position, and so, highlight the problem.

Reply:
I will need to create a new snapshot as the old one is already corrupted in the beginning. Will do

Reply:
I backed up the snapshot and restored to create the other test case! PTAL?